morpheus/utils/downloader.py

```diff
@@ -96,23 +97,20 @@ def get_dask_cluster(self):
         dask_cuda.LocalCUDACluster
         """
 
-        if self._dask_cluster is None:
-            import dask
-            import dask.distributed
+        if Downloader._dask_cluster is None:
             import dask_cuda.utils
 
             logger.debug("Creating dask cluster...")
 
-            # Up the heartbeat interval which can get violated with long download times
-            dask.config.set({"distributed.client.heartbeat": self._dask_heartbeat_interval})
             n_workers = dask_cuda.utils.get_n_gpus()
             threads_per_worker = mp.cpu_count() // n_workers
 
-            self._dask_cluster = dask_cuda.LocalCUDACluster(n_workers=n_workers, threads_per_worker=threads_per_worker)
+            Downloader._dask_cluster = dask_cuda.LocalCUDACluster(n_workers=n_workers,
+                                                                  threads_per_worker=threads_per_worker)
 
-            logger.debug("Creating dask cluster... Done. Dashboard: %s", self._dask_cluster.dashboard_link)
+            logger.debug("Creating dask cluster... Done. Dashboard: %s", Downloader._dask_cluster.dashboard_link)
 
-        return self._dask_cluster
+        return Downloader._dask_cluster
```

Comment:
Crucial: This class can be accessed by multiple `Downloader` instances, potentially running in different threads. We need to synchronize access to creating this cluster with a mutex.